config/routes.rb

```diff
@@ -3,4 +3,14 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  resources :dishes, only: [:show] do
+    resources :dish_ingredients, only: [:create]
```

Comment:
DIshIngredients isn't a child of Dishes, but an Ingredient is. We could refactor these routes in 2 ways:

```ruby
resources :dishes, only: :show do
  resources :ingredients, only: :create, controller: "dish_ingredients"
end 
```

OR 

```ruby
resources :dishes, only: :show
resources :dish_ingredients, only: :create
```